test/parallel/test-cluster-worker-wait-server-close.js

```diff
@@ -22,10 +22,12 @@ if (cluster.isWorker) {
   worker.once('listening', function() {
     net.createConnection(common.PORT, common.localhostIPv4, function() {
       var socket = this;
-      worker.disconnect();
-      setTimeout(function() {
-        socket.write('.');
-        connectionDone = true;
+      setTimeout(function(){
```

Comment:
wfm on linux, what system is this hanging on? why is this timeout needed? putting random timeouts in to solve race conditions is fragile, and in this case, I don't see what is being waited for... the worker is listening, the tcp connection is established... what's the pause for?